configure

```diff
@@ -657,6 +657,10 @@ def configure_node(o):
   elif target_arch in ('mips', 'mipsel'):
     configure_mips(o)
 
+  if flavor in ('aix'):
```

Comment:
Not sure what kind of python it is, but this should be `if flavor in ('aix',):`?